core/model/edition.py

```diff
@@ -126,6 +136,8 @@ class Edition(Base, EditionConstants):
 
     medium = Column(MEDIUM_ENUM, index=True)
 
+    duration = Column(Float, nullable=True)
```

Comment:
I think it would be helpful to add a small comment here with some details of what this number represents. We can probably link out to https://github.com/readium/webpub-manifest/tree/master/contexts/default#duration-and-number-of-pages. It looks like its the float number of seconds in an audiobook.